README.md

```diff
@@ -36,30 +36,14 @@ $config = ['grant_type' => 'client_credentials', 'scope' => 'myscope'];
 
 // Any implementation of PSR-6 Cache will do
 $cache = new \Symfony\Component\Cache\Adapter\FilesystemAdapter();
-$cacheHandler = new \Softonic\OAuth2\Guzzle\Middleware\AccessTokenCacheHandler($cache);
 
-
-$stack = new \GuzzleHttp\HandlerStack();
-$stack->setHandler(new \GuzzleHttp\Handler\CurlHandler());
-
-$addAuthorizationHeader = new \Softonic\OAuth2\Guzzle\Middleware\AddAuthorizationHeader(
+$client = \Softonic\OAuth2\Guzzle\Middleware\ClientBuilder::build(
     $provider,
```

Comment:
What happens if you don't have Oauth in the GraphQL server?

Reply:
Then I would strongly recommend _not_ to use this guzzle middleware :)

Reply:
Hoy are rugby,It would be a has option :-D